modules/ossm-install-cert-manager.adoc

```diff
@@ -0,0 +1,105 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/install/ossm-installing-openshift-service-mesh.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-install-cert-manager_{context}"]
+= Installing cert-manager
+//TP1 content influx. Title, etc may change.
+//Content is very similar to 2.x content
+//all kinds of formatting things to fix. want to see if a build will generate to have a look, and see how it fits structurally with the IA.
+
+You can integrate `cert-manager` with {SMProduct} by deploying `istio-csr` and then creating an `Istio` resource that uses `istio-csr` to process workload and `controlplane` certificate signing requests. This example creates a self signed `Issuer`, but any other `Issuer` can be used instead.
+
+[IMPORTANT]
+====
+The cert-manager tool must be installed before you create and install your `Istio` resource.
+====
+
+.Procedure
+
+. Create the `istio-system` namespace:
```

Comment:
```suggestion
. Create the `istio-system` namespace by running the following command:
```
For accessibility purposes, it's helpful to include the phrase "by running the following command" to make the content clearer for people who are using tools like screen readers to follow this content.